crates/weaver_resolver/src/registry.rs

```diff
@@ -152,9 +153,32 @@ pub fn resolve_semconv_registry(
     );
     check_root_attribute_id_duplicates(&ureg.registry, &attr_name_index, &mut errors);
 
+    gc_unreferenced_objects(registry.manifest(), &mut ureg.registry);
```

Comment:
Can this be optional? I want my registry to be a superset of all otel semconvs and my private semconvs. Alternatively, we'll need a way to succinctly declare the groups to inherit from the dependency.

Reply:
I added a new flag `include_unreferenced` (false by default) to support this use case.